tests/rdma/run.py

```
@@ -14,101 +14,254 @@
 
-    svr = subprocess.Popen(svrcmd, shell=False, stdout=subprocess.PIPE)
+    if not content:
+        return
+
+    print("Valkey Over RDMA valkey-server " + label + " log:")
+    print("---------------\n" + content + "---------------\n")
+
+
+def start_server(cmd, logpath):
+    logfile = open(logpath, "w", buffering=1)
+    svr = subprocess.Popen(cmd, shell=False, stdout=logfile, stderr=subprocess.STDOUT)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)